### ISSUE TITLE
[macOS Big Sur] getpagesize is deprecated in macOS Big Sur

### ISSUE THREAD

**Issue by rbranche** [giampaolo/psutil#1906]
## Summary

* OS: macOS Big Sur
* Architecture: 64bit
* Psutil version: psutil==5.7.3
* Python version: Python 3.8.2
* Type: core

## Description

Even with the previous `unistd.h` include fix (#1791 and #1794), Big Sur has a later POSIX version where `getpagesize` is still not included in `unistd.h`:

```
555 /* Removed in Issue 6 */
556 #if !defined(_POSIX_C_SOURCE) || _POSIX_C_SOURCE < 200112L
557 int  getdtablesize(void) __POSIX_C_DEPRECATED(199506L);
558 int  getpagesize(void) __pure2 __POSIX_C_DEPRECATED(199506L);
559 char    *getpass(const char *) __POSIX_C_DEPRECATED(199506L);
560 #endif
```
---

```
building 'psutil._psutil_osx' extension
clang -Wno-unused-result -Wsign-compare -Wunreachable-code -fno-common -dynamic -DNDEBUG -g -fwrapv -O3 -Wall -iwithsysroot/System/Library/Frameworks/System.framework/PrivateHeaders -iwithsysroot/AppleInternal/Library/Frameworks/Python.framework/Versions/3.8/Headers -arch arm64e -arch x86_64 -DPSUTIL_POSIX=1 -DPSUTIL_SIZEOF_PID_T=4 -DPSUTIL_VERSION=580 -DPSUTIL_OSX=1 -I/AppleInternal/Library/Frameworks/Python.framework/Versions/3.8/include/python3.8 -c psutil/_psutil_common.c -o build/temp.macosx-10.15-x86_64-3.8/psutil/_psutil_common.o
clang -Wno-unused-result -Wsign-compare -Wunreachable-code -fno-common -dynamic -DNDEBUG -g -fwrapv -O3 -Wall -iwithsysroot/System/Library/Frameworks/System.framework/PrivateHeaders -iwithsysroot/AppleInternal/Library/Frameworks/Python.framework/Versions/3.8/Headers -arch arm64e -arch x86_64 -DPSUTIL_POSIX=1 -DPSUTIL_SIZEOF_PID_T=4 -DPSUTIL_VERSION=580 -DPSUTIL_OSX=1 -I/AppleInternal/Library/Frameworks/Python.framework/Versions/3.8/include/python3.8 -c psutil/_psutil_posix.c -o build/temp.macosx-10.15-x86_64-3.8/psutil/_psutil_posix.o
clang -Wno-unused-result -Wsign-compare -Wunreachable-code -fno-common -dynamic -DNDEBUG -g -fwrapv -O3 -Wall -iwithsysroot/System/Library/Frameworks/System.framework/PrivateHeaders -iwithsysroot/AppleInternal/Library/Frameworks/Python.framework/Versions/3.8/Headers -arch arm64e -arch x86_64 -DPSUTIL_POSIX=1 -DPSUTIL_SIZEOF_PID_T=4 -DPSUTIL_VERSION=580 -DPSUTIL_OSX=1 -I/AppleInternal/Library/Frameworks/Python.framework/Versions/3.8/include/python3.8 -c psutil/_psutil_osx.c -o build/temp.macosx-10.15-x86_64-3.8/psutil/_psutil_osx.o
psutil/_psutil_osx.c:51:9: warning: _POSIX_C_SOURCE: 200809L
```

→ `200809L`

`PAGE_SIZE` was added in macOS 10.10 (Yosemite). In fact, `PAGE_SIZE` is already being used in another place in this file already, so it should be ok changing these other `getpagesize` calls to `PAGE_SIZE`.

**Comment by giampaolo** [comment]
This was already fixed in 5.8.0.

**Comment by rbranche** [comment]
Ah, sorry for the noise then.
